app/resources/views/mails/confirmation.blade.php

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
```

Comment:
Your mail changes were merged in a separate pull request.

Please completely remove this commit:
![image](https://user-images.githubusercontent.com/11838191/42116210-4f422524-7bc4-11e8-80ad-7bef96a90f1a.png)

Reply:
@passionInfinite do you know how to remove a commit?  I didn't mean revert it.  If you use "git rebase -i master" and remove the line for the commit, the commit will completely disappear.  Reverting adds another commit.

![image](https://user-images.githubusercontent.com/11838191/42178934-24c31108-7e00-11e8-83ae-205b167f3041.png)